colors/oceanicnext.vim

```diff
@@ -220,6 +220,7 @@ endif
 hi! link FoldColumn LineNr
 hi! link SignColumn LineNr
 call s:HL("ColorColumn", s:none, s:bg2)
+call s:HL("Conceal", s:blue, s:bg)
```

Comment:
Setting the foreground alone suffices here. Background should only be set for items that are:

1. Foundations (like `Normal` = covers entire file buffer, `LineNr` = gutter)
2. Panels (like `PMenu`)
3. Items with explicitly non-default background (like `CursorLine`)

For everything else you'll usually want a `NONE` background, meaning only the text color is overwritten.

Reply:
I inadvertently made the same mistake when linking some existing groups btw, will push a fix soon.